src/features/metadata-instance-editor/Instance.js

```diff
@@ -524,18 +530,41 @@ class Instance extends React.PureComponent<Props, State> {
         return ops;
     }
 
+    /**
+     * Utility function to determine if instance should consider cascading policy
+     *
+     * @return {boolean} true if cascading policy is applicable
+     */
+    isCascadingPolicyApplicable = () => {
```

Comment:
I agree that it's confusing. This new method is appropriately named, as its purpose is to determine whether cascading applies to the metadata instance. The "prop" was initially given a misleading name, as it was used to enable or disable the cascade option, rather than determining its applicability to the instance.

Maybe I should rename the prop to something like `isCascadingPolicyEnabled`